strings/abductee_objectives.txt

```diff
@@ -31,10 +31,10 @@ The CEO of Nanotrasen is coming! Ensure the station is in absolutely pristine co
 Flood the station's powernet with as much electricity as you can.
 Replace all the floor tiles with wood, carpeting, grass or bling.
 You must escape the station! Get the shuttle called!
-Don't allow anyone to be cloned.
+Don't allow anyone to be revived.
```

Comment:
```suggestion
Death should be finality and modern medicine is disrupting the balance. Don't allow anyone to be revived.
```
Context
Also, I doubt yelling is going to be the biggest thing people with this objective are going to do to stop revival.

Reply:
Now that is good flavor text.

Reply:
cobby confirmed to be an abductor